src/pudl/constants.py

```diff
@@ -789,6 +789,7 @@
     'BA': 'Energy Storage, Battery',
     'BT': 'Turbines Used in a Binary Cycle. Including those used for geothermal applications',
     'CA': 'Combined-Cycle -- Steam Part',
+    'CC': 'Combined-Cycle, Total Unit',
```

Comment:
I saw this category when integrating EIA923 from I think 2004. I actually cleaned it out of the data as an error, instead of adding it here, because
* it only occurred on a single plant for a single year
* in subsequent years, that plant broke out into CA and CC parts
* in prior years, that plant left it blank
* The CC category does not appear in EIA docs

Does this category appear more often in some other source, like 860? I'm fine adding it, I just want to know if I should go back and change the 923 stuff to be consistent.

Reply:
I thought that I saw it more than just one time, and that I had seen EIA documentation of it (from which I got the description) but I could be wrong. If what you're recalling is the way it really is, then you definitely did the right thing and the generators will all be given the CA/CT prime mover codes in the entity resolution process.  I need to verify this.

Reply:
I get 715 records when I look for all the CC generators, and they appear fairly consistently (~45 of them) in every year of data from 2004-2020:
```py
(
    pudl_out.gens_eia860()
    .query("prime_mover_code=='CC'")
    .loc[:, [
        "report_date",
        "plant_id_eia",
        "generator_id",
        "prime_mover_code",
    ]]
)
```

Reply:
OK, this seems to be a difference between 923 and 860. I just double checked and 'CC' is not in the prime mover code table of the 923 docs but it is in the 860 table. And 'CC' only appears on a single plant-year in 923 data (generation fuel).

I guess I'll leave everything as is? I think it was an error in the 923, but is fine in the 860.